src/mame/layout/brke23p2.lay

```diff
@@ -0,0 +1,256 @@
+<?xml version="1.0"?>
+<!--
+license:CC0-1.0
+-->
+<mamelayout version="2">
+	<element name="segment">
+		<rect state ="0">
+			<bounds x="0" y="0" width="7" height="7" />
+			<color red="1.0" green="1.0" blue="1.0" />
+		</rect>
+		<rect state ="1">
+			<bounds x="0" y="0" width="7" height="7" />
+			<color red="0.2" green="0.2" blue="0.2" />
+		</rect>
+	</element>
```

Comment:
You don’t need the `bounds` here at all – the element’s internal coordinate system is not related to the coordinate system of the view that uses it.